content/build/bake/matrices.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Matrix targets
+description: Learn how to define and use matrix targets in Bake to fork a single target into multiple different variants
+keywords: build, buildx, bake, buildkit, matrix, hcl, json
+---
+
+A matrix strategy lets you fork a single target into multiple different
+variants, based on parameters that you specify. This works in a similar way to
+[Matrix strategies for GitHub Actions](https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs).
+You can use this to reduce duplication in your bake definition.
```

Comment:
I'm not sure if this should be capitalized, so highlighting in just in case.

```suggestion
You can use this to reduce duplication in your Bake definition.
```